lib/src/loader.ts

```diff
@@ -82,11 +83,37 @@ export function hCaptchaLoader(params: ILoaderParams = { cleanup: true }): Promi
       }
     );
 
-    hCaptchaScripts.push({ promise, scope: frame.window });
-
     return promise;
   } catch (error) {
     sentry.captureException(error);
     return Promise.reject(new Error(SCRIPT_ERROR));
   }
 }
+
+export function loadScript(params, retries = 0) {
+  const message = retries < RETRY_COUNT ? 'Retry loading hCaptcha Api' : 'Exceeded maximum retries';
+
+  const sentry = initSentry(params.sentry);
+
+  return hCaptchaApi(params, sentry).catch(
```

Comment:
@brdlyptrs rejection is not being caught by try/catch, used `.catch()` instead

Reply:
Thats weird, catch should be the same whether this way or vis `try / catch`.